react/fields/FieldLabel/FieldLabel.less

```diff
@@ -0,0 +1,11 @@
+@import (reference) "~seek-style-guide/theme";
+
+.label {
+  @media only screen and (max-width: (@responsive-breakpoint - 1)) {
+    font-weight: @sk-bold;
+  }
+}
+
+.secondaryLabel {
+  font-weight: @sk-light;
```

Comment:
@stephaniemoss when you get a chance can you let us know about this? In this context secondaryLabel is used in TextArea when `Max Character` options is passed in, example at: http://seek-style-guide--53438087bf8ed17f0a3d3cbaacc79035f261ebf2.surge.sh/textarea